readthedocs_ext/_templates/readthedocs-insert.html.tmpl

```diff
@@ -0,0 +1,34 @@
+
+<!-- RTD Extra Head -->
+
+{%- if pagename == "index" %}
+    {%- set canonical_page = "" %}
+{%- elif pagename.endswith("/index") %}
+    {%- set canonical_page = pagename[:-("/index"|length)] + "/" %}
+{%- else %}
+    {%- set ending = "/" if builder == "readthedocsdirhtml" else ".html" %}
+    {%- set canonical_page = pagename + ending %}
+{%- endif %}
+
+<!-- 
+Always link to the latest version, as canonical.
+http://docs.readthedocs.org/en/latest/canonical.html
+-->
+<link rel="canonical" href="{{ canonical_url }}{{ canonical_page }}" />
+
+<script type="text/javascript" src="{{ pathto('_static/readthedocs-data.js', 1) }}"></script>
+
+{% if page_source_suffix %} 		
+<!-- Add page-specific source suffix, which must exist in the page data, not global -->
+<script type="text/javascript">
+READTHEDOCS_DATA['source_suffix'] = {{ page_source_suffix }}
```

Comment:
This should be quoted, and should be changed in the data file template as well. Does this exist here to ensure the change for projects built with a `readthedocs-data.js` but could be out of date?

I added this change to the template in #17 either way.